server/api/store.go

```diff
@@ -394,19 +395,24 @@ func (h *storesHandler) SetAllLimit(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *storesHandler) GetAllLimit(w http.ResponseWriter, r *http.Request) {
-	limit, err := h.GetAllStoresLimit()
+	limits, err := h.GetAllStoresLimit()
 	if err != nil {
 		h.rd.JSON(w, http.StatusInternalServerError, err.Error())
 		return
 	}
-	ret := make(map[uint64]interface{})
-	for s, l := range limit {
-		ret[s] = struct {
-			Rate float64 `json:"rate"`
-		}{Rate: l * schedule.StoreBalanceBaseTime}
+	type LimitResp struct {
```

Comment:
Why do we need to define it?

Reply:
The user may want to know how the value is set, automatically or manually